examples/functions/functions_tester/test/integration/materializeTotalSales.test.js

```diff
@@ -0,0 +1,87 @@
+// :snippet-start: materialize-total-sales-test
+const { app_id } = require("../../realm_config.json");
+const Realm = require("realm");
+const { BSON } = require("realm");
+
+let user;
+const app = new Realm.App(app_id);
+const sandwichId = BSON.ObjectId();
+const saladId = BSON.ObjectId();
+
+// utility function
+async function sleep(ms) {
+  await new Promise((resolve) => setTimeout(resolve, ms));
+}
+
+beforeEach(async () => {
+  const credentials = Realm.Credentials.anonymous();
+  user = await app.logIn(credentials);
+});
+// Clean up. Removes user and data created in the test.
+afterEach(async () => {
+  const db = user.mongoClient("mongodb-atlas").db("store");
+  await db.collection("sales").deleteMany({});
+  await db.collection("total_sales_materialized").deleteMany({});
+  await app.deleteUser(user);
+});
+
+test("Trigger creates a new materialization", async () => {
+  const sales = user
+    .mongoClient("mongodb-atlas")
+    .db("store")
+    .collection("sales");
+
+  await sales.insertOne({
+    _id: BSON.ObjectId(),
+    productId: sandwichId,
+    price: 12.0,
+    timestamp: Date.now(),
+  });
+
+  // give time for trigger to execute
```

Comment:
```suggestion
  // give time for the trigger to execute
```
the nittiest of nits